src/actions/clients.js

```diff
@@ -17,7 +17,9 @@ export const exportProject = createAction(
 );
 export const projectExported = createAction(
   'PROJECT_EXPORTED',
-  (url, exportType) => ({url, exportType}),
+  (url, exportType, projectKey, name) => ({url, exportType, projectKey, name}),
```

Comment:
So adding `projectKey` here is entirely reasonable (although if we’re sticking with positional arguments it seems like it should probably be first?)

However the `name` thing seems pretty specific to repo export—e.g. a gist, snapshot, etc. doesn’t meaningfully have a name.

A couple of alternative options:

1. Just store the URL, and parse it for the information we need when updating. Not _incredibly_ satisfying to have to build in knowledge of GitHub repo URL structure, but it’s not like it’s going to change.
2. Have this action take a `exportData` parameter that can contain an object with any export-type-specific information that might be used elsewhere.

Reply:
went with `exportData`